api/tests/unit/domain/usecases/sessions-mass-import/validate-sessions_test.js

```diff
@@ -171,6 +188,16 @@ describe('Unit | UseCase | sessions-mass-import | validate-sessions', function (
           },
         ];
 
+        sessionsImportValidationService.getUniqueCandidates.onCall(0).returns({
+          uniqueCandidates: [candidate1],
+          duplicateCandidateErrors: [],
+        });
+
+        sessionsImportValidationService.getUniqueCandidates.onCall(1).returns({
```

Comment:
```suggestion
        sessionsImportValidationService.getUniqueCandidates.onSecondCall().returns({
```

Peut être chainé directement aussi au précèdent onCall pour ne pas répéter `sessionsImportValidationService.getUniqueCandidates`

Reply:
Yep ! Corrigé